core/tests/protractor_utils/ExplorationEditorStatsTab.js

```diff
@@ -51,15 +60,17 @@ var ExplorationEditorStatsTab = function() {
 
   this.clickIssue = async function(issueIndex, expectedIssueText) {
     expect(await _getIssueText(issueIndex)).toMatch(expectedIssueText);
-    await _getIssueElement(issueIndex).click();
+    var issueButton = await (
+      _getIssueElement(issueIndex));
+    await action.click(issueIndex.toString(), issueButton);
```

Comment:
Can you make the description (first argument) clearer? This shows up in the error log as something like `{description} is not clickable`, so choose a description that will make that message readable